docs/contributing.rst

```diff
@@ -373,13 +373,13 @@ Geth Fixtures
 
    .. code:: sh
 
-       $ python -m geth.install v1.14.5
+      $ python -m geth.install 1.14.12
```

Comment:
Does this and below need the `v` too?

Reply:
This part is generated with the code where I mentioned the `v` is missing